src/_guides/language/effective-dart/design.md

```diff
@@ -1533,36 +1533,42 @@ The new syntax is a little more verbose, but is consistent with other locations
 where you must use the new syntax.
 
 
-### DO annotate with `Object` instead of `dynamic` to indicate any object is allowed.
+### AVOID using `dynamic` unless you want dynamic dispatch.
 
 Some operations work with any possible object. For example, a `log()` method
 could take any object and call `toString()` on it. Two types in Dart permit all
 values: `Object` and `dynamic`. However, they convey different things. If you
 simply want to state that you allow all objects, use `Object`, as you would in
 Java or C#.
 
-Using `dynamic` sends a more complex signal. It may mean that Dart's type system
-isn't sophisticated enough to represent the set of types that are allowed, or
-that the values are coming from interop or otherwise outside of the purview of
-the static type system, or that you explicitly want runtime dynamism at that
-point in the program.
+The type `dynamic` not only accepts all objects, but it also permits all
+*operations*. Any member access on a value of type `dynamic` is allowed at
```

Comment:
Is "operations" the best term here? Maybe "methods" (or, as the following sentence suggests: "members")?

Reply:
"Methods" is overly precise. "Members" is pretty good, but reads as awkwardly redundant with the next sentence to me. Also, though the paragraph doesn't spell this out, using `dynamic` permits you to pass the value as an argument to a function expecting any type, so there are other "operations" that are allowed outside of member access.

Reply:
Ok, sold (on the use of "operations").